config/locales/en.yml

```diff
@@ -551,6 +551,15 @@ en:
           true: "public"
           false: "private"
         queries: "Queries"
+        status_code: 'Status'
+        status_explanation: 'Project status description'
```

Comment:
Until the required PR is merged into `dev`, the only real diff is the line I'm commenting on.